scripts/validate-dashboards.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -eu
+
+# List of files to exclude
+excludes=( "values.schema.json" "home.json" )
+exculde_string=$(IFS="|"; echo "${excludes[*]}")
+
+# Find valid team names
+github_repo="https://github.com/giantswarm/github.git"
+github_repo_directory="$(mktemp -d -t validate-dashboards.XXXXXXXXXX)"
+trap 'rm -rf "$github_repo_directory"' EXIT
+git clone --depth 1 "$github_repo" "$github_repo_directory"
```

Comment:
will this work in CI?

Reply:
Unsure, but we can change this when we get there.